train.py

```diff
@@ -71,7 +71,8 @@ def main(job_config: JobConfig):
     utils.init_distributed(job_config)
     # initialize GPU memory monitor and get peak flops for MFU calculation
     gpu_memory_monitor = build_gpu_memory_monitor()
-    gpu_peak_flops = utils.get_peak_flops(gpu_memory_monitor.device_name)
+    gpu_peak_flops = utils.get_peak_flops(gpu_memory_monitor)
+    logger.info(f"Peak FLOPS we use for MFU: {gpu_peak_flops} ")
```

Comment:
```suggestion
    logger.info(f"Peak FLOPS used for computing MFU: {gpu_peak_flops} ")
```

Reply:
Do we need to do some formatting of the peak flops, as I can imagine it's a very large number?

Reply:
sure, I can add a format for it.